lib/takeaway.rb

```diff
@@ -0,0 +1,23 @@
+class Takeaway
+  
+  attr_reader :dishes, :order, :price
+
+  def initialize(items)
```

Comment:
items could be delegated to a new Menu class object that holds a hash of dishes + prices